libunsf.c

```diff
@@ -3324,7 +3339,7 @@ UNSF_SYMBOL void unsf_convert_sf_to_gus(UnSF_Options *options) {
                                     if ((sf_num_instrument_indexes * 4 != subchunk.size) ||
                                         (sf_instrument_indexes)) BAD_SF();
 
-                                    sf_instrument_indexes = malloc(sizeof(sfInstBag) * sf_num_instrument_indexes);
+                                    sf_instrument_indexes = (sfInstBag *) malloc(sizeof(sfInstBag) * sf_num_instrument_indexes);
```

Comment:
Dumbass github markup interpreted '-' as a history marker, so my original response
was possibly unreadable. The casts fixes compilation with g++, i.e. fixes the -Wc++-compat
warnings. Do a "make CC=g++" on your unmodified tree to see the errors.